core/src/main/java/com/datastax/oss/driver/internal/core/os/Libc.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.internal.core.os;
+
+import java.util.Optional;
+
+public interface Libc {
+
+  /* Maintained to allow Native.isXAvailable() functionality without trying to make a native call if
+   * the underlying support _is_ available. */
+  public boolean available();
```

Comment:
Nit: we don't specify the `public` modifier for interfaces, it's implied.

Reply:
I've always liked specifying it; I find it easier to have the access specification at hand rather than remembering whether you're looking at an interface or not.  But I'm fine with removing it if that's preferred.